### ISSUE TITLE
TypeError : coercing to Unicode

### ISSUE THREAD

**Issue by t-amorim** [nephila/djangocms-blog#189]
Hello,

First, I want to thank you for your work.

But, unfortunately, I've got an error when I try to add a post or manage config (for example).

```
TypeError at /fr/admin/djangocms_blog/post/add/
coercing to Unicode: need string or buffer, NoneType found
Request Method:	POST
Request URL:	http://site/admin/djangocms_blog/post/add/?language=en
Django Version:	1.7.9
Exception Type:	TypeError
Exception Value:	
coercing to Unicode: need string or buffer, NoneType found
Exception Location:	/var/www/site/env/lib/python2.7/site-packages/django/utils/encoding.py in force_text, line 85
Python Executable:	/usr/local/bin/uwsgi
Python Version:	2.7.6
```

I've found this stackoverflow post that seems to match with the problem : http://stackoverflow.com/questions/16169035/coercing-to-unicode-need-string-or-buffer-nonetype-found-when-rendering-in-dja

### Error during template rendering
```
In template /var/www/site/env/lib/python2.7/site-packages/django/contrib/admin/templates/admin/includes/fieldset.html, error at line 19
coercing to Unicode: need string or buffer, NoneType found
9	            {% for field in line %}
10	                <div{% if not line.fields|length_is:'1' %} class="field-box{% if field.field.name %} field-{{ field.field.name }}{% endif %}{% if not field.is_readonly and field.errors %} errors{% endif %}{% if field.field.is_hidden %} hidden{% endif %}"{% elif field.is_checkbox %} class="checkbox-row"{% endif %}>
11	                    {% if not line.fields|length_is:'1' and not field.is_readonly %}{{ field.errors }}{% endif %}
12	                    {% if field.is_checkbox %}
13	                        {{ field.field }}{{ field.label_tag }}
14	                    {% else %}
15	                        {{ field.label_tag }}
16	                        {% if field.is_readonly %}
17	                            <p>{{ field.contents }}</p>
18	                        {% else %}
19	                            {{ field.field }}
20	                        {% endif %}
21	                    {% endif %}
22	                    {% if field.field.help_text %}
23	                        <p class="help">{{ field.field.help_text|safe }}</p>
24	                    {% endif %}
25	                </div>
26	            {% endfor %}
27	        </div>
28	    {% endfor %}
29	</fieldset>
```

### PIP freeze
```
Django==1.7.9
Django-Select2==4.3.1
Pillow==2.7.0
URLObject==2.4.0
Unidecode==0.04.18
YURL==0.13
aldryn-apphooks-config==0.2.6
aldryn-boilerplates==0.7.3
aldryn-common==1.0.0
aldryn-search==0.2.10
argparse==1.2.1
beautifulsoup4==4.4.0
cmsplugin-filer==1.0.0
dj-database-url==0.3.0
django-admin-enhancer==1.0.0
django-admin-jqueryui==1.8.22
django-admin-sortable==2.0.1
django-appconf==1.0.1
django-appdata==0.1.5
django-background-image==0.0.1
django-blog-zinnia==0.15.2
django-ckeditor==5.0.2
django-classy-tags==0.6.2
django-cms==3.1.2
django-contrib-comments==1.6.1
django-filer==1.0.0.post1
django-haystack==2.4.1
django-hvad==1.3.0
django-inline-ordering==1.0.2
django-meta==0.3.1
django-meta-mixin==0.2.1
django-modeltranslation==0.10.2
django-mptt==0.7.4
django-multilingual-model==0.6
django-parler==1.6
django-polymorphic==0.7.2
django-reversion==1.8.5
django-sekizai==0.8.2
django-simple-math-captcha==1.0.7
django-sortedm2m==1.1.1
django-spurl==0.6.4
django-standard-form==1.1.1
django-tagging==0.4
django-taggit==0.17.6
django-taggit-autosuggest==0.2.8
django-taggit-templatetags==0.2.5
django-templatetag-sugar==1.0
django-treebeard==3.0
django-xmlrpc==0.1.5
djangocms-admin-style==0.2.7
djangocms-apphook-setup==0.1.2
djangocms-blog==0.6.3.post1
djangocms-column==1.5
djangocms-file==0.1
djangocms-flash==0.2.0
djangocms-googlemap==0.2
djangocms-inherit==0.1
djangocms-installer==0.7.8
djangocms-link==1.6.2
djangocms-page-sitemap==0.3.1
djangocms-picture==0.1
djangocms-snippet==1.5
djangocms-style==1.5
djangocms-teaser==0.1
djangocms-text-ckeditor==2.5.2
djangocms-video==0.1
easy-thumbnails==2.2.1
html5lib==0.999
jsonfield==1.0.3
lxml==3.5.0
mots-vides==2015.5.11
psycopg2==2.6.1
pyparsing==2.0.3
pytz==2015.4
six==1.9.0
sorl-thumbnail==12.3
surlex==0.2.0
tzlocal==1.2
uWSGI==2.0.11
wsgiref==0.1.2
zinnia-wysiwyg-ckeditor==1.3
```

Can you help me with that ?

Thank you.

**Comment by yakky** [comment]
@t-amorim I will have a look at this. Can you narrow down the issue to a specific field?

**Comment by t-amorim** [comment]
@yakky Thanks for your help. 
How can I do that ?
It's even happen when I manage config and try to delete the first config I made.

**Comment by yakky** [comment]
@t-amorim what's the value for
``settings.LANGUAGES``, ``settings.CMS_LANGUAGES`` and ``settings.PARLER_LANGUAGES``?

**Comment by yakky** [comment]
(could you try to downgrade parler to 1.4.1?)

**Comment by t-amorim** [comment]
Here's my config.
```
CMS_LANGUAGES = {
    'default': {
        'public': True,
        'hide_untranslated': False,
        'redirect_on_fallback': True,
    },
    1: [
        {
            'public': True,
            'code': 'fr',
            'hide_untranslated': False,
            'name': gettext('fr'),
            'redirect_on_fallback': True,
        },
        {
            'public': True,
            'code': 'en',
            'hide_untranslated': False,
            'name': gettext('en'),
            'redirect_on_fallback': True,
        },
    ],
    2: [
        {
            'public': True,
            'code': 'fr',
            'hide_untranslated': False,
            'name': gettext('fr'),
            'redirect_on_fallback': True,
        },
        {
            'public': True,
            'code': 'en',
            'hide_untranslated': False,
            'name': gettext('en'),
            'redirect_on_fallback': True,
        },
    ],
    3: [
        {
            'public': True,
            'code': 'fr',
            'hide_untranslated': False,
            'name': gettext('fr'),
            'redirect_on_fallback': True,
        },
        {
            'public': True,
            'code': 'en',
            'hide_untranslated': False,
            'name': gettext('en'),
            'redirect_on_fallback': True,
        },
    ],
    4: [
        {
            'public': True,
            'code': 'fr',
            'hide_untranslated': False,
            'name': gettext('fr'),
            'redirect_on_fallback': True,
        },
        {
            'public': True,
            'code': 'en',
            'hide_untranslated': False,
            'name': gettext('en'),
            'redirect_on_fallback': True,
        },
    ],
}
```
LANGUAGES = (
    ('fr', gettext('fr')),
    ('en', gettext('en')),
)
```
```
PARLER_LANGUAGES = {
    1: (
        {'code': 'en',},
        {'code': 'fr',},
    ),
    2: (
        {'code': 'en',},
        {'code': 'fr',},
    ),
    3: (
        {'code': 'en',},
        {'code': 'fr',},
    ),
    4: (
        {'code': 'en',},
        {'code': 'fr',},
    ),
}
```

I downgraded to django-parler==1.4 but no changes, I still have the same error.

**Comment by yakky** [comment]
@t-amorim thanks, I'm trying to reproduce as much as possible your configuration

**Comment by t-amorim** [comment]
@yakky Thanks you !

**Comment by yakky** [comment]
@t-amorim install parler==1.6 and add this to ``PARLER_LANGUAGES``::

        'default': {
            'fallbacks': ['en', 'fr'],
        }

This should fix your issue

**Comment by t-amorim** [comment]
@yakky It's amazing how you can find so fast the solution... 
Thanks a lot it was that !

**Comment by yakky** [comment]
@t-amorim great!
Will add more information to the README to document this

**Comment by yakky** [comment]
Fixed in e1c63e4
